keps/sig-network/20190603-endpointslices/README.md

```diff
@@ -626,6 +678,173 @@ address types.
 3. Deprecate the `IP` address type, making it invalid for new EndpointSlices in
 1.17 before becoming fully invalid in 1.18.
 
+## Production Readiness Review Questionnaire
+
+### Feature enablement and rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - *EndpointSlice*: Guards controller and API server enablement.
+    - *EndpointSliceProxying*: Guards kube-proxy enablement
+
+* **Does enabling the feature change any default behavior?**
+  No.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we rollback
+  the enablement)?**
+  Yes.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+  Reenabling the `EndpointSlice` feature gate could result in the controller
+  cleaning up or updating stale EndpointSlices. Reenabling the
+  `EndpointSliceProxying` feature gate should not result in any changes to API
+  resources.
```

Comment:
Though it may result in reprogramming iptables (or whatever else is used) if/whenever Endpoints and EndpointSlices are not in-sync.

Reply:
Good catch, added.